### ISSUE TITLE
[Suggestion]: Add a means of viewing beacon recipes without JEI

### ISSUE THREAD

**Issue by GearsawStudios** [nonamecrackers2/crackers-wither-storm-mod#1645]
### Description of feature request

If you've looked around in the Discord, the amount of messages asking for instructions to summon the Symbiont is unreal. It's gotten to the point where there is a message pinned on how to do it. To me, that's a sign that perhaps it's not obvious how to use the beacon. For a mod that claims to be vanilla styled, it's quite ironic it uses another mod to complete itself. While JEI support is a fantastic thing in its own right, relying on it for basic knowledge isn't great design, especially since most people use this mod alone or with performance mods.

Essentially, this mod isn't complete unless another mod is used. For something as polished and standalone as this mod (even coming with its own renderer,) it doesn't make sense to rely so much on another mod. It's only exemplified by how many recipes the beacon has, so it only makes sense to add the final thing that makes a crafting station a crafting station.

Fixing this is simple, there evidently is code being used for the JEI version, repurpose that code and incorporate it into a beacon recipe book button that can be added to the beacon. If the Easter Egg(s?, I struggle to use it without JEI) are a problem, simply hide their recipes.

**Comment by Laughingcat27** [comment]
good idea, and perhaps some of the easter egg recipes could be found in hidden books in the bowels

**Comment by BluePineappleGuy** [comment]
> ### Description of feature request
> If you've looked around in the Discord, the amount of messages asking for instructions to summon the Symbiont is unreal. It's gotten to the point where there is a message pinned on how to do it. To me, that's a sign that perhaps it's not obvious how to use the beacon. For a mod that claims to be vanilla styled, it's quite ironic it uses another mod to complete itself. While JEI support is a fantastic thing in its own right, relying on it for basic knowledge isn't great design, especially since most people use this mod alone or with performance mods.
> 
> Essentially, this mod isn't complete unless another mod is used. For something as polished and standalone as this mod (even coming with its own renderer,) it doesn't make sense to rely so much on another mod. It's only exemplified by how many recipes the beacon has, so it only makes sense to add the final thing that makes a crafting station a crafting station.
> 
> Fixing this is simple, there evidently is code being used for the JEI version, repurpose that code and incorporate it into a beacon recipe book button that can be added to the beacon. If the Easter Egg(s?, I struggle to use it without JEI) are a problem, simply hide their recipes.

like some sort of rune tablet or smth, that would be cool

**Comment by Draco1246** [comment]
Bro I didn't even know the beacon had any other recipes than to summon the storm
